### ISSUE TITLE
game-01, 

### ISSUE THREAD

**Issue by sanchez201316196** [papaya-holdings/jigsaw#6]
Que tal, estoy por enviar las respuestas en un Drive que solo se puede ingresar por enlace,
pero tengo una consulta con respecto a la pregunta 1 quisiera saber si el par de números que indican deben ser consecutivos del array que ingreso, debido a que el ejemplo lo muestra así.

Gracias !

**Comment by CristianLlanos** [comment]
Hola, @sanchez201316196, no es necesario que sean consecutivos. Pero sí deben cumplir con la suma.

**Comment by CristianLlanos** [comment]
No te olvides de seguir las [recomendaciones](https://github.com/papaya-holdings/jigsaw#once-you-are-finished) sobre cómo debes enviarnos tus respuestas :)

**Comment by sanchez201316196** [comment]
Claro, e leído, las recomendaciones sobre subir a un repositorio publico, pero no sé si quizás otras personas se encuentren postulando a la oferta laboral y puedan ver estos issues y ver mi repositorio. Por eso quiero saber si estaría bien subirlo en un mi Drive de Google.

**Comment by sanchez201316196** [comment]
Hola, qué tal. Ayer envié el enlace publico como lo solicitaron, pero subí una versión de prueba del Gane-01, deje unas notas en los commits de la pregunta 1, si lo podrían considerar.

**Comment by sanchez201316196** [comment]
Ya realice el cambio con una actualización. 😁
